app/controllers/visns_controller.rb

```diff
@@ -73,6 +73,10 @@ def show
 
   def load_facilities_show_rows
     @facilities = VaFacility.get_by_visn(@visn).select(:common_name, :id, :official_station_name, :slug, :station_number, :fy17_parent_station_complexity_level)
+    @clinical_resource_hubs = ClinicalResourceHub.cached_clinical_resource_hubs
+    visn_id = Visn.where(number: params["number"].to_i).first.id
+    @clinical_resource_hubs = @clinical_resource_hubs.where(visn_id: visn_id)
```

Comment:
Same here. I think we can use `@visn_clinical_resource_hub` or `@clinical_resource_hub` and `.find_by`, since there's only one `clinical_resource_hub` per visn.

Reply:
Problem with find_by is it turns var in to a clinical_resource_hub object versus and active record object... so appending it to the @facilities object doesn't work on the next line.   We can take a further look at this if you want.

Reply:
So I tried it out, and we could do `@facilities + Array(ClinicalResourceHub.find_by(visn_id: visn_id))`, which may be better performance-wise, since the only reason we are calling a `.where()` here is to turn it into an array.